### PR TITLE
init canvas project (pending)

### PR DESCRIPTION
**Pull request by FabianCristancho**
# Zero To Mastery - Canvas 2022 🎉🎉

👉 _**Please remove the below and replace with your own values, leaving the headers where they are.**_ 👈

## PR contains:
- INFO ABOUT YOUR PULL REQUEST GOES HERE (Please be as descriptive as possible) 🤜

## Breaking changes
- IF ANYTHING YOU'RE COMMITTING WOULD BREAK SOMETHING, INCLUDE HERE WHAT WOULD BREAK
- IF YOU HAVE NO BREAKING CHANGES, ENTER 'None'

## Screenshots
- IF YOU HAVE ANY SCREENSHOTS, INCLUDE THEM HERE.
- IF YOU HAVE NO SCREENSHOTS, ENTER 'None'

### CODE REVIEW

File: src/art/FabianCristancho/script.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
+var canvas = document.getElementById('jack');
+var context = canvas.getContext('2d');
+// context.fillStyle = '#6666ff';
```
Comment (sophiabrandt) [inline src/art/FabianCristancho/script.js:3]: Please remove outcommented code.

File: src/art/FabianCristancho/style.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+.viewport {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100%;
+}
+
+/* .workspace {
```
Comment (sophiabrandt) [inline src/art/FabianCristancho/style.css:16]: Same here.

File: src/art/FabianCristancho/script.js (hunk 1 above)
Comment (FabianCristancho) [reply src/art/FabianCristancho/script.js]: thanks for the clarification. I just deleted the commented lines, so please check it :D

File: src/art/FabianCristancho/style.css (hunk 2 above)
Comment (FabianCristancho) [reply src/art/FabianCristancho/style.css]: thanks for the clarification. I just deleted the commented lines, so please check it :D

**Pull request merged**